std/StringTools.hx

```diff
@@ -413,6 +413,15 @@ class StringTools {
 		return false;
 		#end
 	}
+	
+	/**
+		Converts the first character of the String `s` to upper case and the 
+		remaining characters to lower case.
```

Comment:
I find that really surprising to the point that it's probably better to not add the method at all. In all my use-cases for it I never wanted the tail string to be modified and I can't really see in which scenarios this makes practical sense. Let's leave this one out of the standard library.